yoshi-library: build command throws if deps are installed via yarn

I created a library and installed its dev/deps via yarn, then I ran `yarn build` and got the following error, seems like the module resolving is not working properly:

```sh
$ yoshi-library build

Found 0 type errors.

/Users/*/node_modules/yoshi-flow-library/build/cli.js:15
    throw error;
    ^

Error: Cannot find module '@babel/plugin-transform-typescript' from '/Users/*'
    at Function.resolveSync [as sync] (/Users/*/node_modules/resolve/lib/sync.js:81:15)
    at resolveStandardizedName (/Users/*/node_modules/yoshi-common/node_modules/@babel/core/lib/config/files/plugins.js:101:31)
    at resolvePlugin (/Users/*/node_modules/yoshi-common/node_modules/@babel/core/lib/config/files/plugins.js:54:10)
    at loadPlugin (/Users/*/node_modules/yoshi-common/node_modules/@babel/core/lib/config/files/plugins.js:62:20)
    at createDescriptor (/Users/*/node_modules/yoshi-common/node_modules/@babel/core/lib/config/config-descriptors.js:154:9)
    at /Users/*/node_modules/yoshi-common/node_modules/@babel/core/lib/config/config-descriptors.js:109:50
    at Array.map (<anonymous>)
    at createDescriptors (/Users/*/node_modules/yoshi-common/node_modules/@babel/core/lib/config/config-descriptors.js:109:29)
    at createPluginDescriptors (/Users/*/node_modules/yoshi-common/node_modules/@babel/core/lib/config/config-descriptors.js:105:10)
    at /Users/*/node_modules/yoshi-common/node_modules/@babel/core/lib/config/config-descriptors.js:63:53
    at cachedFunction (/Users/*/node_modules/yoshi-common/node_modules/@babel/core/lib/config/caching.js:62:27)
    at cachedFunction.next (<anonymous>)
    at evaluateSync (/Users/*/node_modules/gensync/index.js:244:28)
    at sync (/Users/*/node_modules/gensync/index.js:84:14)
    at plugins (/Users/*/node_modules/yoshi-common/node_modules/@babel/core/lib/config/config-descriptors.js:28:77)
    at mergeChainOpts (/Users/*/node_modules/yoshi-common/node_modules/@babel/core/lib/config/config-chain.js:319:26) {
  code: 'MODULE_NOT_FOUND'
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Works OK if installation is via npm.

--- a/packages/yoshi-common/src/typescript/run-babel.ts
+++ b/packages/yoshi-common/src/typescript/run-babel.ts
@@ -27,8 +27,8 @@ function transpileFile({ filePath, cwd }: { filePath: string; cwd: string }) {
     filename: filePathInBuildDir,
     sourceFileName: relativeSourceFileName,
     sourceMaps: true,
-    plugins: ['@babel/plugin-transform-typescript'],
-    presets: [['babel-preset-yoshi', { mode: 'test' }]],
+    plugins: [require.resolve('@babel/plugin-transform-typescript')],
+    presets: [[require.resolve('babel-preset-yoshi'), { mode: 'test' }]],
   });
 
   const filePathInBuildDirNoExtensions = stripExtension(filePathInBuildDir);

